eva/executor/orderby_executor.py

```diff
@@ -65,14 +73,30 @@ def exec(self) -> Iterator[Batch]:
             aggregated_batch_list.append(batch)
         aggregated_batch = Batch.concat(aggregated_batch_list, copy=False)
 
+        # Column can be a functional expression, so if it
+        # is not in columns, it needs to be re-evaluated.
+        merge_batch_list = [aggregated_batch]
+        for col in self._columns:
+            if isinstance(col, TupleValueExpression):
+                col_name_list = [col.col_alias]
+            elif isinstance(col, FunctionExpression):
+                col_name_list = col.col_alias
+
+            for col_name in col_name_list:
+                if col_name not in aggregated_batch.frames:
+                    batch = col.evaluate(aggregated_batch)
+                    merge_batch_list.append(batch)
+        if len(merge_batch_list) > 1:
+            aggregated_batch = Batch.merge_column_wise(merge_batch_list)
+
         # sorts the batch
         try:
             aggregated_batch.sort_orderby(
                 by=self.extract_column_names(),
                 sort_type=self.extract_sort_types(),
             )
         except KeyError:
-            # pass for now
+            # raise ExecutorError(str(e))
```

Comment:
Uncomment

Reply:
You mean we want to raise Error here? If so, this test case (https://github.com/georgia-tech-db/eva/blob/90ed6f2ea1d2e7f8b2016c7f9bfc696746338980/test/integration_tests/test_select_executor.py#L61) will fail.

Reply:
I thought the new code you added will also fix it.

Reply:
You are right.